tsconfig.json

```diff
@@ -9,5 +9,10 @@
     "removeComments": false,
     "noImplicitAny": true,
     "suppressImplicitAnyIndexErrors": true
-  }
+  },
+  "exclude": [
```

Comment:
[tsconfig.json](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html)
_"The "exclude" property defaults to excluding the node_modules, bower_components, jspm_packages and <outDir> directories when not specified."_